src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/model/Accesstoken.java

```diff
@@ -0,0 +1,123 @@
+package didentity.amos.digitalIdentity.model;
```

Comment:
Answers, Responses and tokens should be saved in another folder. /model/ folder is just for database entities



An example would be /messages/ or something like this